moped-database/migrations/1657565379012_alter_table_public_moped_proj_components_add_column_interim_project_compoennt_id/up.sql

```diff
@@ -0,0 +1,2 @@
+alter table "public"."moped_proj_components" add column "interim_project_component_id" integer
```

Comment:
The DB is configured to only store integers, which feels correct. The front-end is a bit looser, and will let you submit numbers, and sometimes they throw a toast error about the the graphql endpoint not accepting it, and sometimes they do and silently fail to save it. Here are some example inputs that give results:

`24601` → Green toast, works fine
`24.601` → 
<img width="652" alt="image" src="https://user-images.githubusercontent.com/3653206/178807933-ba4cd268-9e20-4b86-a6cf-9ca96ddf6c08.png">
`2460e1` → Similar red toast error
`2.4e601` -> Green, but doesn't save a value, which is confusing to me.

Reply:
This reminds me of the issue that I had with project funding amounts. I made a component called `FundingAmountIntegerField` that might be helpful here? Feel free to make that component name more general to integer inputs if they end up aligning closely.